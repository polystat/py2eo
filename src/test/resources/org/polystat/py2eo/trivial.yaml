python: |
  def testFib1():
      f0 = 0
      f1 = 1
      ii = 0
      while ii < 10:
          f2 = f0 + f1
          ii = ii + 1
          f0 = f1
          f1 = f2
      return 55 == f0