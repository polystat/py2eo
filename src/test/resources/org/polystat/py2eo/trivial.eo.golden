+package org.eolang
+alias org.eolang.txt.sprintf
+alias org.eolang.io.stdout
+junit

[] > trivial
  memory > bogusForceDataize
  seq > @
    seq
      [] > testFib1
        memory > f1
        memory > f0
        memory > i
        memory > f2
        seq > @
          seq
            f0.write 0
            f1.write 1
            i.write 0
            while.
              ((i).less 10)
              [unused]
                seq > @
                  seq
                    ^.f2.write ((^.f0).add (^.f1))
                    ^.i.write ((^.i).add 1)
                    ^.f0.write (^.f1)
                    ^.f1.write (^.f2)
            stdout (sprintf "%d\n" (f0))
      bogusForceDataize.write (((testFib1)))
